.rubocop_todo.yml

```diff
@@ -227,6 +229,8 @@ RSpec/MultipleExpectations:
 # Configuration parameters: AllowSubject.
 RSpec/MultipleMemoizedHelpers:
   Max: 20
+  Exclude:
```

Comment:
Same here

Reply:
@luke-hill Many of the memoized helpers (at least`test_id`, `step_id`, `name`) only exist to be used once in the constructor, and never referenced again. I think the number of `let`s could be cut down dramatically by introducing some factory methods, something like `#build_test_case` and `#build_test_step`.

Again, though, this would be a pretty big refactor, diff-wise, for the file, so I'd like to do it on a separate follow-up PR that builds on-top of this. Proceed?

Reply:
@luke-hill Also realizing that switching `C::C::Test::Case#initialize` to kwargs would improve this situation all by itself, because many variables are only in `let`s just so that they're named somewhere, and aren't bare `double`s with no context. With kwargs, it would be a non-issue to just use e.g. `Test::Case.new(test_id: double, name: double, ... )` etc, I think.

Reply:
Sounds like a plan, happy for it to be done later. Just floating out ideas as you mentioned you'd be keen on doing some additional work whilst I try to crack the nut that is the CCK compliance for v12.